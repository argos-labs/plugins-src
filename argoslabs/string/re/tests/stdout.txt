C:\Users\satto3\Desktop\Temp\Plugin Test\