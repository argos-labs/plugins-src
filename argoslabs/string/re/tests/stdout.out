C:\Users\sato3\Desktop\cinnabar-2.csv